pkg/apis/extensions/types.go

```diff
@@ -525,6 +525,29 @@ type JobList struct {
 	Items []Job `json:"items"`
 }
 
+// JobTemplate describes a template for creating copies of a predefined pod.
+type JobTemplate struct {
```

Comment:
(Of course this should get moved to pkg/apis/batch/types.go in the near future, but I guess it's ok here now)